client/components/embed-container/index.jsx

```diff
@@ -21,10 +21,10 @@ const embedsToLookFor = {
 const cacheBustQuery = `?v=${ Math.floor( new Date().getTime() / ( 1000 * 60 * 60 * 24 * 10 ) ) }`; // A new query every 10 days
 
 const SLIDESHOW_URLS = {
-	CSS: `https://s0.wp.com/wp-content/mu-plugins/shortcodes/css/slideshow-shortcode.css${ cacheBustQuery }`,
-	CYCLE_JS: `https://s0.wp.com/wp-content/mu-plugins/shortcodes/js/jquery.cycle.min.js${ cacheBustQuery }`,
-	JS: `https://s0.wp.com/wp-content/mu-plugins/shortcodes/js/slideshow-shortcode.js${ cacheBustQuery }`,
-	SPINNER: `https://s0.wp.com/wp-content/mu-plugins/shortcodes/img/slideshow-loader.gif${ cacheBustQuery }`,
+	CSS: `https://s0.wp.com/wp-content/mu-plugins/jetpack-plugin/production/modules/shortcodes/css/slideshow-shortcode.css${ cacheBustQuery }`,
+	CYCLE_JS: `https://s0.wp.com/wp-content/mu-plugins/jetpack-plugin/production/modules/shortcodes/js/jquery.cycle.min.js${ cacheBustQuery }`,
+	JS: `https://s0.wp.com/wp-content/mu-plugins/jetpack-plugin/production/modules/shortcodes/js/slideshow-shortcode.js${ cacheBustQuery }`,
```

Comment:
I noticed that there are a couple of lines slightly different between `jetpack-plugin/production/modules/shortcodes/js/slideshow-shortcode.js` and `mu-plugins/shortcodes/js/slideshow-shortcode.js` (additional content in the jetpack-plugin version). It actually looks like the jetpack-plugin version is more complete, and the mu-plugins version is missing that content.
Not sure if this is anything that needs follow-up - testing using Calypso live the slideshow shortcode worked just fine.

Reply:
Thanks for having a look. The Jetpack version is more complete, and since it's just adding aria labels it should be fine 👍